ts/screens/profile/ProfileMainScreen.tsx

```diff
@@ -252,6 +255,20 @@ class ProfileMainScreen extends React.PureComponent<Props> {
               </View>
             </ListItem>
 
+            <ListItem>
+              <View style={styles.debugModeSection}>
+                <View>
+                  <Text style={styles.itemLeftText}>PagoPA QA ENV</Text>
```

Comment:
move this string to locales, also the text should be something like `PagoPA Test`

```suggestion
                  <Text style={styles.itemLeftText}>PagoPA Test</Text>
```